featurebyte/query_graph/sql/tiling.py

```diff
@@ -328,11 +328,11 @@ def tile(self, col: Optional[InputColumn], agg_id: str) -> list[TileSpec]:
         sum_expression = expressions.Anonymous(
             this="VECTOR_AGGREGATE_SUM", expressions=[quoted_identifier(col.name)]
         )
+        count = expressions.Count(this=expressions.Star())
+        cast_as_float = expressions.Cast(this=count, to="FLOAT")
```

Comment:
Would it be safer to use `expressions.DataType.build("DOUBLE")` for the destination type? In spark the float type is smaller and might overflow.

Reply:
sure will update